### ISSUE TITLE
if "config" is run from a repo dir not named exactly "berrybrew", env var warning thrown

### ISSUE THREAD

**Issue by stevieb9** [stevieb9/berrybrew#170]
Example:

```
git clone ... berrybrew_prod
cd berrybrew_prod
bin\berrybrew config
```

Output:

```
Failed to add berrybrew to path
```

...however, it was actually inserted properly.

**Comment by stevieb9** [comment]
After looking at this, I think we'll need to add a new file in the data directory to contain the directory we originally ran `config` out of. This way, we can read that in and correctly check it instead of using the `berrybrew\bin` regex.

The `off` feature also uses this directory name, so we have to stash it someplace.

**Comment by stevieb9** [comment]
Path management was addressed in caa20e1888e0c8e433597ff0730f19cba876b4d3. Will verify that the issue is actually resolved, then mark it as such.

**Comment by stevieb9** [comment]
Confirmed fixed in caa20e1888e0c8e433597ff0730f19cba876b4d3 (v1.29)
